src/content/en/updates/2019/02/using-twa.md

```diff
@@ -442,6 +442,119 @@ adb logcat | grep -e OriginVerifier -e digital_asset_links
 With the upload APK generated, you can now [upload the app to the Play
 Store](https://developer.android.com/studio/publish/upload-bundle).
 
+## Adding a Splash Screen {: #making-a-splash }
+
+Starting on **Chrome 75**, Trusted Web Activities are have support for Splash Screens.
+The Splash Screen can be added by adding a few new image files and configurations to the
+project.
+
+Make sure to update to **Chrome 75 or above** and use the
+[latest version of TWA Support Library](https://jitpack.io/#GoogleChrome/custom-tabs-client).
+
+### Generating the images for the Splash Screen {: #create-images }
+Android devices can have different [screen sizes](https://developer.android.com/training/multiscreen/screensizes)
+and [pixel densities](https://developer.android.com/training/multiscreen/screendensities).
+To ensure the Splash Screen looks good on all devices, you will need to generate
+the image for each pixel density.
+
+A full explanation of [display-independent pixels (dp or dip)](https://developer.android.com/training/multiscreen/screendensities#TaskUseDP) is beyond the scope of this article, but a good
+starting point is to create an image that is 300x300dp, which is equivalent to 300x300
```

Comment:
I'm not sure we should recommend 300x300 specifically, especially if the logos are "flat" enough that having a rectangular shape makes more sense. We can use 300x300 dp just as an example, and say that it means about 2x2 inches on device with any screen density. We may pick 320x320 instead to make it exaclty 2x2.

Reply:
Changed the text to say it's an example and to use a 320x320 image. Added added that it makes a 2x2in image on screen.